File: fizzbuzz.rb
for x in 1..100 do
    if (x % 15 == 0)
        puts "FizzBuzz"
    elsif (x % 3 == 0)
        puts "Fizz"
    elsif (x % 5 == 0)
        puts "Buzz"
    elsif (x % 3 && x % 5)
        puts x
    end
end

File: fizzbuzz.java
class Fizzbuzz
{
    public static void main(String[] args)
    {
        int i;

        i = 0;
        while (++i <= 100)
        {
            if (i % 3 == 0)
                System.out.printf("Fizz");
            if (i % 5 == 0)
                System.out.printf("Buzz");
            if (i % 3 != 0 && i % 5 != 0)
                System.out.printf("%d", i);
            System.out.println();
        }
    }
}
